db/migrate/20160427110629_create_service_interactions.rb

```diff
@@ -0,0 +1,12 @@
+class CreateServiceInteractions < ActiveRecord::Migration
+  def change
+    create_table :service_interactions do |t|
+      t.integer :service_identifier
+      t.string :service_label
```

Comment:
Do we need to add `service_label` and `interaction_label` here? I would assume that these would always be the label of the referred service/interaction? (knowing full well what they say about 'assume' :) ). So just wondering if we can just keep the identifiers and get rid of the labels...